tests/model/test_bugscache.py

```diff
@@ -156,3 +156,28 @@ def get_search_query(sql):
 
         search_query = get_search_query(context.captured_queries[-1]['sql'])
         assert search_query
+
+
+SEARCH_TERMS = (
+    ("(test_popup_preventdefault_chrome.xul+)", " test_popup_preventdefault_chrome.xul  "),
+    (
+        "TEST-UNEXPECTED-TIMEOUT | /webrtc/promises-call.html | Can set up a basic WebRTC call with only data using promises. - Test timed out",
+        "TEST UNEXPECTED TIMEOUT | /webrtc/promises call.html | Can set up a basic WebRTC call with only data using promises.   Test timed out",
+    ),
+    (
+        "*command timed out: 3600 seconds without output running~",
+        " command timed out: 3600 seconds without output running ",
+    ),
+    (
+        "\"input password unmask.html#abc_def 0 7 7 7\"",
+        " input password unmask.html#abc_def 0 7 7 7 ",
```

Comment:
I don't know how many test cases is considered sufficient, but this last one was the edge case that hit the mysql bug.

Reply:
I think this is fine for this case - it's a regression test to make sure we don't hit the specific issue again.